tests/integration/linux_bridge_test.py

```diff
@@ -147,6 +147,21 @@ def bond0(port0_up):
         yield bond0
 
 
+def _vlan_filtering_enabled(bridge_name):
+    _, vlan_filtering_flag, _ = exec_cmd(
+        cmd=(
+            "nmcli",
+            "-g",
+            "bridge.vlan-filtering",
+            "connection",
+            "show",
+            bridge_name,
+        ),
+        check=True,
+    )
+    return vlan_filtering_flag.strip() == "yes"
```

Comment:
Instead of checking `nmcli` output, please try use `npc` or `ip  -d  link show br0` which are  checking the kernel state instead of on-disk configurations.

Reply:
Done, thanks!